.github/workflows/cleanup-repo.yaml

```diff
@@ -1,7 +1,7 @@
 ﻿name: "Cleanup Repo"
 on:
   schedule:
-    - cron: '0 7 * * *'
+    - cron: '0 8 * * *'
   workflow_dispatch: {}
```

Comment:
<!-- Consider confirming the reasoning behind changing the cron schedule from 7 AM to 8 AM. Is this in line with the expected workflow timing? -->